verifier/verifier.go

```diff
@@ -898,3 +903,135 @@ func logVerificationResult(logger log.Logger, result *notation.ValidationResult)
 func isRequiredVerificationPluginVer(pluginVer string, minPluginVer string) bool {
 	return semver.Compare("v"+pluginVer, "v"+minPluginVer) != -1
 }
+
+// verifyTimestamp provides core verification logic of authentic timestamp under
+// signing scheme `notary.x509`.
+func verifyTimestamp(ctx context.Context, policyName string, trustStores []string, signatureVerification trustpolicy.SignatureVerification, x509TrustStore truststore.X509TrustStore, r revocation.Revocation, outcome *notation.VerificationOutcome) error {
+	logger := log.GetLogger(ctx)
+
+	signerInfo := outcome.EnvelopeContent.SignerInfo
+	performTimestampVerification := true
+
+	// check if tsa trust store is configured in trust policy
+	tsaEnabled, err := isTSATrustStoreInPolicy(policyName, trustStores)
```

Comment:
Where is this method defined?

Reply:
It's defined in [`verify/helpers.go`](https://github.com/Two-Hearts/notation-go/blob/de827c159518087de2066eb2d4b325ba714d8969/verifier/helpers.go#L174) along with load trust store helper functions.